client/src/Movies/Movie.js

```diff
@@ -26,42 +29,27 @@ export default class Movie extends Component {
       });
   };
   // Uncomment this code when you're ready for the stretch problems
-  // componentWillReceiveProps(newProps){
-  //   if(this.props.match.params.id !== newProps.match.params.id){
-  //     this.fetchMovie(newProps.match.params.id);
-  //   }
-  // }
+  componentWillReceiveProps(newProps){
```

Comment:
Just to take note, componentWillReceiveProps is deprecated. Don't use it in your own project.